### PR TITLE
Add warning message if consultation is still ongoing

### PR DESCRIPTION
**Pull request by benbaumann95**
### Story Link

https://trello.com/c/awbPSSDk/1842-bt-warn-assessor-reviewer-if-recommendation-is-before-21-days-publicity-time-expired

### Screenshots

Screenshot 2023-08-07 at 14 39 44

### CODE REVIEW

File: spec/models/consultation_spec.rb [hunk 1]
```diff
@@ -138,4 +138,37 @@
       expect(consultation.neighbour_responses_by_summary_tag).to eq({ "objection" => 1, "supportive" => 2, "neutral" => 1 })
     end
   end
+
+  describe "publicity_active?" do
```
Comment (benjamineskola) [inline spec/models/consultation_spec.rb:142]: Would a test for the behaviour when the end date is today be useful?

Comment (benbaumann95) [reply spec/models/consultation_spec.rb:142]: I think we'd still want this warning if the end date is today as technically it should go on until the end of day right

Comment (benjamineskola) [reply spec/models/consultation_spec.rb:142]: Yeah, I think the behaviour seems correct: "greater than today".

Comment (benbaumann95) [reply spec/models/consultation_spec.rb:142]: Added a spec to compare the actual day rather than time in this instance.

**Pull request merged**